### DISCUSSION TITLE
LIS compile error on JULES 5.0 (using user.cfg)

### DISCUSSION THREAD

**Discussion by jachungithub** [NASA-LIS/LISF#1035] [Q&A] [answered]
Hi,

I have a bunch of error messages to compile LIS (***LISF_public_release_7.3.3***) using ***user.cfg*** (so that "***JULES 5.0***" turns on). Most of the error messages are
> "../surfacemodels/land/jules.5.0/jules50_lsmMod.F90(98): error #7002: Error in opening the compiled module file.  Check INCLUDE paths."  (please see the attached file (***error message.txt***) for more details)

The following files are also attached:
> configure.lis
user.cfg
LIS_misc.h
LIS_NetCDF_inc.h
LIS_plugins.h

I believe that I followed the direction in the LIS users guideline:
> 1. Creating a ***user.cfg*** 
> 2. Cleaning up the ***make*** directory
> 3. Re-running the ***configure*** script
> 4. Compiling

Any suggestions would be appreciated.

[lis error.zip](https://github.com/NASA-LIS/LISF/files/8389597/lis.error.zip)

**Comment by emkemp** [comment, marked as answer]
Hi @jachungithub 

The JULES LSM is not included in LIS, as that software is closed source.  The LIS code only includes an interface to that LSM.

You must download that separately from the UK Met Office, after getting permission from them to do so.  Special instructions for coupling JULES to LIS are given in https://github.com/NASA-LIS/LISF/blob/support/lisf-public-7.3/lis/surfacemodels/land/jules.5.0/COUPLING_INSTRUCTIONS.adoc

**Reply by jachungithub** [reply]
Thanks a lot!
